skimage/registration/__init__.py

```diff
@@ -1,8 +1,44 @@
+"""
+Image registration module
```

Comment:
Maybe we can make this line describe what image registration is? What do you think about
```suggestion
"""Utilities to relate different sets of data in one coordinate system.
```